sympy/ntheory/tests/test_ecm.py

```diff
@@ -0,0 +1,4 @@
+from sympy.ntheory import ecm
+
+assert ecm(3146531246531241245132451321) == {3, 100327907731, 10454157497791297}
+assert ecm(46167045131415113) == {43, 2634823, 407485517}
```

Comment:
Add more tests.

Reply:
I have a ques. As we use random in ecm, smichr told me to use `seed` in the test cases. I am not sure how to use that.

Reply:
Moreover, the main advantage of ecm is seen in bigger numbers >30digits and the timing of those test will be atleast 5~10secs and can be even more depending on the size.

Reply:
Currently i have added the test cases and it required around ~15-25secs to evaluate.

Reply:
Yes because the seed will create the same random numbers on multiple executions of the code. You can look at it from [here](https://www.geeksforgeeks.org/random-seed-in-python/).

If these tests take much time than you can add @slow tag to these tests.